pkg/kubelet/kubelet_pods.go

```diff
@@ -215,14 +215,15 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 			}
 		}
 
-		// Docker Volume Mounts fail on Windows if it is not of the form C:/
+		// Docker Volume Mounts fail on Windows if it is not of the form C:/ nor a named pipe starting with \\.\pipe\
```

Comment:
This would turn `\\server\share` into `c:\\server\share` today. If moby supports SMB paths, than can you adjust this so it won't prepend `c:` in front of SMB paths?